test/e2e/instrumentation/monitoring/stackdriver.go

```diff
@@ -27,8 +27,9 @@ import (
 
 	"github.com/onsi/ginkgo"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/kubernetes/test/e2e/common"
+	e2ecommon "k8s.io/kubernetes/test/e2e/common"
```

Comment:
Generally agree with this change but can we undo this one and `e2emetrics`, think it's better to keep the PR only changing e2elog imports.

Reply:
fixed now